README.md

```diff
@@ -0,0 +1,54 @@
+|-------------------------|:-----------------------------------:|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `pam_ldap_nslcd_conf`  | `{'uid': 'nslcd', 'gid': 'nslcd' }` | Dict containing the option key/value pairs. According to [the nslcd.conf man page](https://linux.die.net/man/5/nslcd.conf). If a value can be defined multiple times just use a list containg all values. |
+| `pam_ldap_access_conf` | `[]`                                | List containing Dicts with access settings. According to [the access.conf man page](https://linux.die.net/man/5/access.conf).                                                                             |
+
+### `pam_ldap_access_conf`
+Each list entry has to have following attributes
+| Name         | Required/Default   | Description                                                                                                                                                                                                                                                                                                                                              |
+|--------------|:------------------:|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `permission` | :heavy_check_mark: | Can be either a "+" character (plus) for access granted or a "-" character (minus) for access denied.                                                                                                                                                                                                                                                    |
+| `object`     | :heavy_check_mark: | The users/group field, should be a list of one or more login names, group names, or ALL (which always matches). To differentiate user entries from group entries, group entries should be written with brackets, e.g. (group).                                                                                                                           |
+| `origins`    | :heavy_check_mark: | The origins field, should be a list of one or more tty names (for non-networked logins), host names, domain names (begin with "."), host addresses, internet network numbers (end with "."), internet network addresses with network mask (where network mask can be a decimal number or an internet address also), ALL (which always matches) or LOCAL. |
```

Comment:
Unnecessary comma in first sentence
`.` twice
`ALL`
`LOCAL`
do you mean CIDR by decimal number?

Reply:
this is taken from the man page